packages/mobile/src/components/Files/ChatFileOverview.tsx

```diff
@@ -174,8 +211,22 @@ export const ChatFileOverview = memo(
       });
       if (medias.didCancel) return;
 
+      const newAssets: ImageSource[] =
+        medias.assets?.map((value) => ({
+          height: value.height || 0,
+          width: value.width || 0,
+          name: value.fileName,
+          type: value.type && value.type === 'image/jpg' ? 'image/jpeg' : value.type,
+          uri: value.uri,
+          filename: value.fileName,
+          date: Date.parse(value.timestamp || new Date().toUTCString()),
+          filepath: value.originalPath,
+          id: value.id,
+          fileSize: value.fileSize,
+        })) || [];
+
       // Keep assets without a type out of it.. We're never sure what it is...
```

Comment:
The comment doesn't really match the code here anymore.. But should the comment be removed or the code adjusted?